pydantic/__init__.py

```diff
@@ -32,9 +32,10 @@
     # dataclasses
     'dataclasses',
     # decorators
+    'field_validator',
+    'model_validator',
     'root_validator',
     'validator',
-    'field_validator',
```

Comment:
I put the four validator decorators in alphabetical order, which also conveniently puts the v2 validators first.